rp235x-hal-examples/Cargo.toml

```diff
@@ -33,6 +33,9 @@ panic-halt = "0.2.0"
 pio = "0.2.0"
 pio-proc = "0.2.0"
 rp235x-hal = {path = "../rp235x-hal", version = "0.3.0", features = ["binary-info", "critical-section-impl", "rt", "defmt"]}
+# The examples use features not yet available in rp-binary-info 0.1.0,
+# so the minimum version implied by the rp235x-hal 0.3.0 dependency is not sufficient.
+rp-binary-info = "0.1.1"
```

Comment:
I guess this also needs a path = ?

Reply:
Indeed. Would be fine without path once 0.1.1 is released, but for now path = is required.